src/components/AccountView.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { View, StyleSheet, ScrollView } from "react-native";
+
+const AccountView = () => {
+  return <ScrollView></ScrollView>;
+};
```

Comment:
Why is there an empty component here?

Reply:
I needed to setup the account page to enable the navigation between the pages. As it is only sprint 2, I haven't filled it in yet.